cg/services/post_processing/pacbio/run_file_manager/run_file_manager.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+from cg.constants import FileExtensions
+from cg.constants.pacbio import PacBioDirsAndFiles
+from cg.services.post_processing.abstract_classes import RunFileManager
+from cg.services.post_processing.exc import PostProcessingFileNotFoundError
+from cg.services.post_processing.pacbio.run_data_generator.run_data import PacBioRunData
+from cg.services.post_processing.validators import validate_files_exist
+from cg.utils.files import get_files_matching_pattern
+
+
+class PacBioRunFileManager(RunFileManager):
+
+    def get_files_to_parse(self, run_data: PacBioRunData) -> list[Path]:
```

Comment:
I'm thinking we could make this methods receive the full path instead of the whole RunData object @ChrOertlin

Reply:
if that is all it needs, sure!

Reply:
Will evaluate in another PR